backend/docs/api/inventory_devices_v1.yaml

```diff
@@ -0,0 +1,170 @@
+openapi: 3.0.1
+info:
+  title: Device inventory
+  description: |
+    An API for uploading device attributes. Intended for use by devices.
+
+    Devices can upload vendor-specific attributes (software/hardware info, health checks, metrics, etc.) of various data types to the backend.
+  version: "1"
+servers:
+- url: https://hosted.mender.io/api/devices/v1/inventory
+paths:
+  /device/attributes:
+    put:
+      tags:
+      - Device API
+      summary: Replace the set of attributes for a device
+      description: |
+        Replaces the attribute set for the authenticated device with the provided one.
+        The device ID is retrieved from the authorization header.
+
+        This method replaces all the attributes with the new set:
+
+        * attributes not provided in the set are removed from the db
+
+        * the values of existing attributes are overwritten
+
+        * attributes assigned for the first time are automatically created
+      operationId: Replace Attributes
+      requestBody:
+        description: A list of attribute descriptors.
+        content:
+          application/json:
+            schema:
+              title: ListOfAttributes
+              type: array
+              example:
+              - name: ip_addr
+                value: 1.2.3.4
+                description: IP address
+              - name: mac_addr
+                value: 00.01:02:03:04:05
+                description: MAC address
+              items:
+                $ref: '#/components/schemas/Attribute'
+        required: true
+      responses:
+        "200":
+          description: Attributes were uploaded successfully.
+          content: {}
+        "401":
+          description: The device is not authenticated.
+          content: {}
+        "400":
+          description: Missing/malformed request parameters or body.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "500":
+          description: Internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+      - DeviceJWT: []
+      x-codegen-request-body-name: attributes
```

Comment:
Do we need to keep these tags in the specs?

Reply:
no; code generators are a bit annoying sometimes...
removed